homeworks/03-data-structures/src/binary-search-tree.test.ts

```diff
@@ -0,0 +1,40 @@
+import {BinarySearchTree, BinarySearchTreeImpl} from "./binary-search-tree"
+import {node, TraverseType, TreeNode} from "./tree-common"
+
+test('binary search tree - has', () => {
```

Comment:
[code style] inconsistent use of quotes, either use double quotes everywhere or single quotes

Reply:
Normally I would use ES Lint and fix that, but here it seemed besides the point.